content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx

```diff
@@ -823,7 +823,7 @@ It is now impossible for two people to book the same seat:
 3. Seat 3A claimed by Sorcha (`version` is incremented to 1, booking succeeds)
 4. Seat 3A claimed by Ellen (in-memory `version` (0) does not match database `version` (1) - booking does not succeed)
 
-### Interactive Transactions (in Preview)
+### Interactive Transactions
```

Comment:
Still unrelated, but why dropping the (Preview) label here? In preparation for the GA of interactive transactions?

Reply:
Yes, have i jumped the gun on that? Looking through my notes i can see that its not mentioned for 7th sept release so i think i may have made a mistake here.